frontend/dialogs/OBSBasicSourceSelect.cpp

```diff
@@ -15,100 +16,70 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ******************************************************************************/
 
+#include "OBSApp.hpp"
```

Comment:
```suggestion
#include <OBSApp.hpp>
```

The included file is not in the same directory as the including file, so this needs to be resolved using include paths.